### DISCUSSION TITLE
Mobile Devices: Video Playback Issue with clipStartTime and clipEndTime on Chrome and Safari

### DISCUSSION THREAD

**Discussion by yoshitsugu-tahara** [vidstack/player#1195] [Q&A]
**Environment**

- Device: iPhone 15
- OS: iOS 17.3.1
- Browser: Google Chrome, Safari
- Library Version: Vidstack v1.10.9(Remix)

**Issue Description**

After implementing video trimming using `clipStartTime` and `clipEndTime` properties, the video fails to play on mobile devices in Google Chrome and Safari browsers. The issue does not occur on desktop browsers, where video playback works as expected.

Removing or commenting out `clipStartTime` and `clipEndTime` resolves the issue, and the video plays correctly on mobile devices.

**Error Message**

When inspecting the issue on a mobile device using Safari's Web Inspector, the following error message is encountered:

```
webkit blob resource error 1
```

Could you please help me understand the cause of this issue and suggest a possible solution? Thank you for your assistance.

**Comment by mihar-22** [comment]
I'm not noticing any issues in testing, if you can provide a [repro](https://github.com/vidstack/player/blob/main/.github/REPRO.md) that would be awesome. Thanks!
